Test dir-dangling-entry.sh never finishes, therefore,
we disable it to ensure that the test-suite terminates.

Not suitable for upstream.

--- texinfo-6.1/info/Makefile.am	2021-05-19 09:55:04.319490440 +0000
+++ texinfo-6.1/info/Makefile.am	2021-05-19 09:54:38.211456633 +0000
@@ -106,7 +106,6 @@ TESTS = \
 	t/dir-3-menus.sh \
 	t/dir-menus-sloppily.sh \
 	t/where-dir-file.sh \
-	t/dir-dangling-entry.sh \
 	t/dir-entry-to-subdir.sh \
 	t/spec-file-node.sh \
 	t/spec-no-node.sh \
